packages/pages/src/common/src/template/internal/getTemplateFilepaths.ts

```diff
@@ -21,7 +22,9 @@ export const getTemplateFilepaths = (paths: Path[]): string[] => {
   const templateFilepaths: string[] = [];
   const addedFilenames: Set<string> = new Set();
   paths.forEach((p) => {
-    const filepaths = globSync(`${p.getAbsolutePath()}/*.{tsx,jsx,js,ts}`);
+    const filepaths = globSync(
+      convertToPosixPath(`${p.getAbsolutePath()}/*.{tsx,jsx,js,ts}`)
```

Comment:
There's a slash here. Is it working ok?

Reply:
Yes, globSync() requires a Posix path (even on Windows).
Just flipping the slashes is ok though, so it can accept a path like C:/root/myLocation/tacos.exe
https://stackoverflow.com/questions/55512907/node-glob-sync-returns-empty-array